prepare-operator-metadata.yml

```diff
@@ -21,20 +21,54 @@
     operator_work_dir: "/home/jenkins/agent/test-operator"
     operator_dir: "/tmp/test"
     work_dir: "{{ lookup('env', 'WORKSPACE') }}"
-    jq_bin_path : "/usr/bin/jq"
+    jq_bin_path: "/usr/bin/jq"
     yq_bin_path: "/usr/local/bin/yq"
     testing_bin_path: "/usr/local/bin"
-    current_channel: '' # Added to avoid a bug with undefined variables
+    # Added to avoid a bug with undefined variables
+    current_channel: ''
 
   tasks:
-    
+
     - name: "Set variables for operator_work_dir and operator_dir"
       set_fact:
-        operator_work_dir: "{{ operator_work_dir | default('/home/jenkins/agent/test-operator')}}"
+        operator_work_dir: "{{ operator_work_dir |
+                               default('/home/jenkins/agent/test-operator')}}"
         operator_dir: "{{ operator_dir | default('/tmp/test') }}"
+        parse_operator_metadata_results: {'result': '', 'msg': ''}
+
+    - block:
+        - name: "Run operator-courier nest to copy operator metadata in nested format to the work dir"
+          shell: "operator-courier nest {{ operator_dir }} {{ operator_work_dir }}"
+
+        - name: "Parse operator metadata needed to run the tests"
```

Comment:
The parse_operator_metadata role is already included after the `operator-courier nest` block, so I don't think it should be included as part of the block here.

Reply:
The `parse_operator_metadata` role also sets the parse_operator_metadata_results and writes to the `parse_operator_metadata_results.json` - won't that conflict with this block?

Reply:
@dirgim No, it will not. 
It will only overwrite the existing contents of the parse_operator_medata_data_results.json. 
It's more like updating its contents. Rather than, conflicting.